WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeViewController.swift

```diff
@@ -166,6 +171,12 @@ extension AddAttributeViewController: UITableViewDataSource {
 extension AddAttributeViewController: UITableViewDelegate {
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         tableView.deselectRow(at: indexPath, animated: true)
+
+        let row = viewModel.sections[indexPath.section].rows[indexPath.row]
+        guard row == .existingAttribute else {
+            return
+        }
+        presentAddAttributeOptions(for: viewModel.localAndGlobalAttributes[indexPath.row])
```

Comment:
nit: just wondering, might there be other select actions in this screen? if so, it might be worth using a switch statement:

```suggestion
        switch row {
            case . existingAttribute:
                presentAddAttributeOptions(for: viewModel.localAndGlobalAttributes[indexPath.row])
            default:
                return
        }
```

Reply:
For the moment there is only this action since the other action is connected to the button "Next"  :)